docs/configuration/backends/kubernetes.md

```diff
@@ -161,6 +161,7 @@ The following security annotations are applicable on the Ingress object:
 | `ingress.kubernetes.io/public-key:VALUE`                 | Adds pinned HTST public key header.                                                                                                                                                                 |
 | `ingress.kubernetes.io/referrer-policy:VALUE`            | Adds referrer policy  header.                                                                                                                                                                       |
 | `ingress.kubernetes.io/is-development:false`             | This will cause the `AllowedHosts`, `SSLRedirect`, and `STSSeconds`/`STSIncludeSubdomains` options to be ignored during development.<br>When deploying to production, be sure to set this to false. |
+| `ingress.kubernetes.io/app-root:VALUE`             |  Redirects all requests for / to the defined path. Non-root paths will not be affected by this annotation and handled normally. |
```

Comment:
this is not the right place to add this documentation, please move this in the section above.